src/main/java/com/softserve/academy/spaced/repetition/DTO/impl/CourseCommentDTO.java

```diff
@@ -0,0 +1,26 @@
+package com.softserve.academy.spaced.repetition.DTO.impl;
+
+import com.softserve.academy.spaced.repetition.DTO.DTO;
+import com.softserve.academy.spaced.repetition.domain.CourseComment;
+import com.softserve.academy.spaced.repetition.domain.Person;
+import org.springframework.hateoas.Link;
+
+import java.util.Date;
+
+public class CourseCommentDTO extends DTO<CourseComment> {
```

Comment:
I propose to merge CourseCommentDTO with DeckCommentDTO and make one class
`public class CommentDTO extends DTO<Comment> { ... }`
They are transfering the same data to UI. In object-oriented programming objects should be reused for less copy-paste.